### DISCUSSION TITLE
Can I change the data structure of the submitted event

### DISCUSSION THREAD

**Discussion by zhushenhang** [react-hook-form/react-hook-form#4757] [Q&A]
```   
const  onSubmit = (value) => { }
```
Click the submit button, the default value is an object. 
like this 
```
const value = { 
  a : "1111",
  b : "22222",
  a : "3333"
}
```
I hope to define the data structure myself when submitting the data, because I may have two identical keys, and the latter will overwrite the former

like this 
```
const value = { 
  outputs: [ 
    { a : "1111" , b: "22222"}
    { a : "3333" , b: "4444"}
  ]
}
```

Thanks!

**Comment by apoorvmote** [comment]
Your output looks like an array field. Have you tried that https://react-hook-form.com/api/usefieldarray
